src/main/java/org/jabref/gui/commonfxcontrols/SaveOrderConfigPanel.java

```diff
@@ -71,9 +71,9 @@ private void initialize() {
     }
 
     public void changeExportDescriptionToSave() {
-        exportInOriginalOrder.setText(Localization.lang("Save entries in their original order"));
-        exportInSpecifiedOrder.setText(Localization.lang("Save entries ordered as specified"));
-        exportInTableOrder.setText(Localization.lang("Save in current table sort order"));
+        exportInOriginalOrder.setText(Localization.lang("Keep original order"));
+        exportInSpecifiedOrder.setText(Localization.lang("Use specified order"));
+        exportInTableOrder.setText(Localization.lang("Use current table sort order"));
     }
```

Comment:
This method and the call to can be removed entirely.

Reply:
Good catch! Otherwise it's good from my side,tested it locally